.github/workflows/labels.yml

```diff
@@ -0,0 +1,31 @@
+name: Import labels
+on:
+  push:
+    branches:
+      - main
```

Comment:
Should this be running on every commit to main? Or would it be better to use the `workflow_dispatch` trigger and run this manually?

Reply:
Yes, ideally it would be great if we could filter with `if: ${{ <expression> }}`. But I think that could come later.

Or like you suggest, it could be a manual trigger. But tbh I prefer automated if possible, even if not currently efficient

Reply:
I could use this but not sure if it `OR` or `AND` with the branch name?

```yml
on:
  push:
    paths:
    - 'docs/labels.json'
```

Reply:
I guess I can test it in the branch

Reply:
It is `OR`

Reply:
I'm more curious why this needs to run more than once, to be honest. It imports the labels to the repository, but then the labels aren't going to change so why does it need to import again?

Reply:
If there are merged changes to the `labels.json` file, it will be great for it to automatically update, one less thing to do / forget.

I filtered it, only to do the import when that `json` file is changed.

<img width="708" alt="Screenshot 2021-03-02 at 18 19 26" src="https://user-images.githubusercontent.com/624760/109695676-16011000-7b84-11eb-96ac-172b5055abff.png">

using

```yml
- name: check label file for change
        uses: pheel/path-watcher-action@v1
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          paths: 'docs/labels.json'
      - if: steps.modified.outputs.modified
```

https://github.com/eddiejaoude/stargate/runs/2015790089?check_suite_focus=true